doc/release-notes/0.33/0.33.md

```diff
@@ -22,21 +22,22 @@
 * Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
 -->
 
-# Eclipse OpenJ9 version 0.33.0 release notes
+# Eclipse OpenJ9 version 0.33.x release notes
 
-These release notes support the [Eclipse OpenJ9 0.33.0 release plan](https://projects.eclipse.org/projects/technology.openj9/releases/0.33.0/plan).
+These release notes support the [Eclipse OpenJ9 0.33.0 release plan](https://projects.eclipse.org/projects/technology.openj9/releases/0.33.0/plan)
+and the [Eclipse OpenJ9 0.33.1 release plan](https://projects.eclipse.org/projects/technology.openj9/releases/0.33.1/plan).
 
 ## Supported environments
 
-OpenJ9 release 0.33.0 supports OpenJDK 8, 11, 17, 18.
+OpenJ9 release 0.33.x supports OpenJDK 8, 11, 17, 18.
```

Comment:
Like in eclipse-openj9/openj9-docs#979, this should say "releases 0.33.x support".

Reply:
I think we settled on "release supports" so leaving it as-is.